sdk/eventhub/eventhubs-checkpointstore-tables/test/tables-checkpointstorespec.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { TableCheckpointStore } from "../src";
+
+if (typeof TableCheckpointStore == "function") {
+  console.log("TableCheckpointstore is a function");
+} else {
+  console.log("Error: TableCheckpointstore is not a function");
+}
```

Comment:
We can write a test to ensure we've got our test framework set up correctly. Something like this:
```ts
import chai from "chai";
const should = chai.should();
import { TableCheckpointStore } from "../src";

describe("TableCheckpointStore", () => {
  it("is exported from the package", () => {
    should.exist(TableCheckpointStore, "Expected TableCheckpointStore to be exported.");
  });
});
```